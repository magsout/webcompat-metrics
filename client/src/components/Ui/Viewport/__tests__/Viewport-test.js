/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import Viewport from "../";

it("renders Viewport default correctly", () => {
  const tree = renderer
    .create(
      <Viewport>
        <span>content</span>
      </Viewport>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
