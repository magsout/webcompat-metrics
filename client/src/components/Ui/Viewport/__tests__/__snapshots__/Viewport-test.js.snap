// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Viewport default correctly 1`] = `
<div
  className="component"
>
  <span>
    content
  </span>
</div>
`;
